src/devServer/HTTPServer.ts

```diff
@@ -74,6 +77,16 @@ Development server running http://localhost:${port} @ ${packageInfo.version}
         return socketServer;
     }
 
+    private createServer(): http.Server | https.Server {
```

Comment:
I almost forgot...
Can you write tests for that?

Reply:
Wrote a test for the https option... tbh i'm a bit lost in the test directory. I don't seem to find anywhere where the HTTPserver is tested

Reply:
I don't think there was any.
It's looks fine for me.
Let's wait @nchanged advice.